pkgs/development/python-modules/pynacl/pynacl-no-timeout-and-deadline.patch

```diff
@@ -0,0 +1,49 @@
+diff --git a/tests/test_pwhash.py b/tests/test_pwhash.py
+index 9634c85..7f20316 100644
+--- a/tests/test_pwhash.py
++++ b/tests/test_pwhash.py
+@@ -20,7 +20,7 @@ import os
+ import sys
+ import unicodedata as ud
+ 
+-from hypothesis import given, settings
++from hypothesis import given, settings, unlimited
+ from hypothesis.strategies import integers, text
+ 
+ import pytest
+@@ -411,7 +411,7 @@ def test_str_verify_argon2_ref_fail(password_hash, password):
+        integers(min_value=1024 * 1024,
+                 max_value=16 * 1024 * 1024)
+        )
+-@settings(deadline=1500, max_examples=20)
++@settings(timeout=unlimited, deadline=None, max_examples=20)
```

Comment:
When there was no timeout before, why do you have to set it to unlimited?

Reply:
Because it's set to 60s by default: http://hypothesis.readthedocs.io/en/latest/settings.html?highlight=timeout#hypothesis.settings.timeout

I put a time.sleep into the tests and not specifying the timeout actually lets it fail if it takes longer than 60s. Of course you need a really really slow system for this to take longer than a minute.

Reply:
If that's the case, what do you think about disabling the timeout for the hypothesis package completely, @FRidh? Of course I don't know if that's possible.

Reply:
Besides that the timeout feature is also already deprecated and will be removed at some point in the future: http://hypothesis.readthedocs.io/en/latest/changes.html#id118
Until then using unlimited is recommended, though.